samples/nrf_desktop/src/modules/usb_state.c

```diff
@@ -221,13 +240,20 @@ static void device_status(enum usb_dc_status_code cb_status, const u8_t *param)
 	}
 }
 
+static void protocol_change(u8_t protocol)
+{
+	hid_protocol = protocol;
+	LOG_INF("Protocol changed to: %s %s", protocol ? "REPORT" : "BOOT", "_PROTOCOL");
```

Comment:
`LOG_INF("Protocol changed to: %s_PROTOCOL", protocol ? "REPORT" : "BOOT");`
or simply
``LOG_INF("%s_PROTOCOL selected", protocol ? "REPORT" : "BOOT");``
as it will use less flash

Reply:
done